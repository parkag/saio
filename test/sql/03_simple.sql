load 'saio';

set saio_seed to 0.5;
set enable_mergejoin to false;
set enable_nestloop to false;

explain (costs off) select * from
	tenk1 t1,
	tenk1 t2,
	tenk1 t3
where
	t1.thousand = t3.thousand and
	t3.hundred = t2.hundred;
