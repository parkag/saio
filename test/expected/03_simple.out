load 'saio';
set saio_seed to 0.5;
set enable_mergejoin to false;
set enable_nestloop to false;
explain (costs off) select * from
	tenk1 t1,
	tenk1 t2,
	tenk1 t3
where
	t1.thousand = t3.thousand and
	t3.hundred = t2.hundred;
                   QUERY PLAN                   
------------------------------------------------
 Hash Join
   Hash Cond: (t3.hundred = t2.hundred)
   ->  Hash Join
         Hash Cond: (t1.thousand = t3.thousand)
         ->  Seq Scan on tenk1 t1
         ->  Hash
               ->  Seq Scan on tenk1 t3
   ->  Hash
         ->  Seq Scan on tenk1 t2
(9 rows)

